src/installer/corehost/cli/fxr/static/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Given the upcoming change: https://github.com/dotnet/runtime/issues/1154#issuecomment-619130049
We could consider not building hostfxr_shared.lib etc, but simply building static_apphost.exe directly.
Again: This is just for consideration ... may simplify the build logic, not sure it is a good idea.